examples/nlp/masked_language_modeling.py

```diff
@@ -117,7 +119,7 @@ def get_data_from_text_files(folder_name):
 train_df = get_data_from_text_files("train")
 test_df = get_data_from_text_files("test")
 
-all_data = train_df.append(test_df)
+all_data = train_df._append(test_df)
```

Comment:
I wouldn't use a protected method like `_append`. It looks like the official replacement for `append` is `concat`.